### PR TITLE
[Ubuntu] move apt pkgs to the toolset.json

### PR DESCRIPTION
**Pull request by al-cheb**
# Description
Changes:
- move jq installation to the apt.sh
- move apt package list to the toolset.json from basic.sh
- move docker images from containercache.sh to the docker-moby.sh
- remove containercache.sh script
- remove some sections from packer templates

#### Related issue:
https://github.com/actions/virtual-environments-internal/issues/1080
## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by maxim-lobanov — APPROVED** [review]

File: images/linux/scripts/installers/basic.sh [hunk 1]
```diff
@@ -4,90 +4,13 @@
...
-
-apt-get install -y --no-install-recommends $libcurelVer
-
-# install additional packages only for Ubuntu16.04
-if isUbuntu16; then
-    common_packages="$common_packages
-            libicu55"
-fi
 
+toolsetJson="$INSTALLER_SCRIPT_FOLDER/toolset.json"
```
Comment (AlenaSviridenko) [inline images/linux/scripts/installers/basic.sh:11]: Maybe rename to `toolset`? I think it is not important to specify that it is json

File: images/linux/scripts/installers/docker-moby.sh [hunk 2]
```diff
@@ -49,19 +49,28 @@ else
...
-docker pull node:12-alpine
-docker pull debian:8
-docker pull debian:9
-docker pull alpine:3.7
-docker pull alpine:3.8
-docker pull alpine:3.9
-docker pull alpine:3.10
-docker pull ubuntu:14.04
+# Pull images
+images=(
```
Comment (AlenaSviridenko) [inline images/linux/scripts/installers/docker-moby.sh:53]: does it make sense to move docker images to toolset too?

File: images/linux/scripts/installers/basic.sh (hunk 1 above)
Comment (al-cheb) [reply images/linux/scripts/installers/basic.sh:11]: @AlenaSviridenko, Should we rename toolsetJson -> toolset in android.sh and azpowershell.sh too?

File: images/linux/scripts/installers/docker-moby.sh (hunk 2 above)
Comment (al-cheb) [reply images/linux/scripts/installers/docker-moby.sh:53]: cc @maxim-lobanov

File: images/linux/scripts/installers/basic.sh (hunk 1 above)
Comment (AlenaSviridenko) [reply images/linux/scripts/installers/basic.sh:11]: @al-cheb why not? Looks like an easy update

Comment (al-cheb) [reply images/linux/scripts/installers/basic.sh]: Done.

**Review by vsafonkin — APPROVED** [review]

File: images/linux/scripts/installers/docker-moby.sh (hunk 2 above)
Comment (maxim-lobanov) [reply images/linux/scripts/installers/docker-moby.sh:53]: I like @AlenaSviridenko's suggestion but I don't think that it should be in scope of this PR.
Could you please create work item on internal board to improve docker installation?

**Pull request merged**